src/energyplus/ForwardTranslator.cpp

```diff
@@ -402,6 +402,16 @@ namespace energyplus {
       }
     }
 
+    // remove orphan Generator:WindTurbine
+    for (auto& wt : model.getConcreteModelObjects<GeneratorWindTurbine>()) {
+      if (!wt.electricLoadCenterDistribution()) {
+        LOG(Warn,
+            "GeneratorWindTurbine " << wt.name().get() << " is not referenced by any ElectricLoadCenterDistribution, it will not be translated.");
+        wt.remove();
+        continue;
+      }
+    }
```

Comment:
So I had a similar realization in https://github.com/NREL/OpenStudio/pull/4194/commits/949fe92542a7ec28122c6fecb511a3e226a74335.

This stuff shouldn't exist. The children of ELCD (Generators, etc) should not be added to the list of objects to always translate. The ElectricLoadCenterDistribution only should be added to that list, and it should be responsible to trigger translation of its generators (and others) instead. 

I'm pretty sure this is just an historical side effect to please the sketchup plugin.